ipalib/util.py

```diff
@@ -68,6 +64,19 @@
 if six.PY3:
     unicode = str
 
+
+try:
+    # pylint: disable=import-error,ipa-forbidden-import
+    from ipaplatform.paths import paths
```

Comment:
I'd rather hard-code the constants here and have a unit test that verifies that the hard-coded values are equal to the values in ``ipaplatform.paths``. We want to ensure that values stay in sync.

Reply:
But that would disallow changing paths on other platform then the hard-coded one. Basically breaking the purpose of ipaplatform.

Reply:
It is not supported to change these paths on any platform any way. Changing the paths would break PyPI packages. You have to make a decision for one of these three options:

* Don't allow any platform to override ``IPA_CLIENT_SYSRESTORE`` and ``IPA_DEFAULT_CONF``
* Ban ``pylint: disable=ipa-forbidden-import`` and don't use it anywhere. Instead move the function out of ``ipalib`` and into ``ipalib.install``.
* Ship ``ipaplatform`` as a PyPI package and change ``ipaplatform`` to detect the platform at import time. I had a patch that detected the platform from ``/etc/system-release`` but it was refused by Petr and Honza.

Reply:
Beware of moving functions as integrators may rely on them.